src/Arduino_WiFiConnectionHandler.h

```diff
@@ -71,18 +67,12 @@ class WiFiConnectionHandler : public ConnectionHandler {
 
     bool keepAlive;
 
-    OnNetworkEventCallback  _on_connect_event_callback,
-                            _on_disconnect_event_callback,
-                            _on_error_event_callback;
-
-    static void execNetworkEventCallback(OnNetworkEventCallback & callback, void * callback_arg);
-
-    NetworkConnectionState update_handleInit         ();
-    NetworkConnectionState update_handleConnecting   ();
-    NetworkConnectionState update_handleConnected    ();
-    NetworkConnectionState update_handleGetTime      ();
+    NetworkConnectionState update_handleInit();
```

Comment:
Formatting change ... of brace formatting. Makes the change harder to read. Please restore.